enacts/enactstozarr.py

```diff
@@ -48,53 +36,45 @@ def set_up_dims(xda):
     # and look into xESMF.
     #
     # [1] https://climatedataguide.ucar.edu/climate-data-tools-and-analysis/regridding-overview
-    rr_mrg = rr_mrg.interp(
-        X=np.arange(rr_mrg.X.min(), rr_mrg.X.max() + RESOLUTION, RESOLUTION),
-        Y=np.arange(rr_mrg.Y.min(), rr_mrg.Y.max() + RESOLUTION, RESOLUTION),
-    )
-
-rr_mrg = rr_mrg.chunk(chunks=CHUNKS)
-
-xr.Dataset().merge(rr_mrg).to_zarr(
-  store = RR_MRG_ZARR_PATH
-)
-
-if not os.access(CONFIG["data_dir"], os.W_OK | os.X_OK):
-    sys.exit("can't write to output directory")
-
-def set_up_dims(xda):
-    datestr = Path(xda.encoding["source"]).name.split("_")[2]
-    year = int(datestr[0:4])
-    month = int(datestr[4:6])
-    dekad = int(datestr[6:7])
-    day = (dekad - 1) * 10 + 1
-    xda = xda.expand_dims(T = [dt.datetime(year, month, day)])
-    xda = xda.rename({'Lon': 'X','Lat': 'Y'})
-    return xda
+        print("Your data will be regridded. Refer to function documentation for more information on this.")
+        data = data.interp(
+            X=np.arange(data.X.min(), data.X.max() + resolution, resolution),
+            Y=np.arange(data.Y.min(),data.Y.max() + resolution, resolution),
+        )    
+    return data
 
 def convert(variable):
-    netcdf = list(sorted(Path(CONFIG['data_src'][variable]).glob("*.nc")))
-
+    print(f"converting files for: {variable}")
+    
+    var_name = CONFIG['vars'][variable][2]    
+    nc_path = f"{CONFIG['nc_path']}{CONFIG['vars'][variable][0]}"
+    netcdf = list(sorted(Path(nc_path).glob("*.nc")))
+    
     data = xr.open_mfdataset(
         netcdf,
         preprocess = set_up_dims,
         parallel=False
-    )#[variable]
-    print(data)
+    )[var_name]
+    if ZARR_RESOLUTION != 0:
```

Comment:
So I believe if you literally define it as 
null
in the config file, then it becomes None when you read the config file and assigne to ZARR_RESOLUTION and I assume that's probably better than using 0 (even though 0 is a good choice since it's impossible to have such a resolution but still).

Reply:
Ah yes thank you, I was confused as to why defining it as `None` was not working..